dali/python/nvidia/dali/pipeline.py

```diff
@@ -1168,7 +1187,7 @@ def outputs(self, cuda_stream=None):
             e.g. ``cupy.cuda.Stream``, ``torch.cuda.Stream``
             The stream to which the returned `TensorLists` are bound.
             Defaults to None, which means that the outputs are synchronized with the host.
-            Works only with pipelines constructed with `experimental_exec_dynamic=True`.
+            Works only with pipelines constructed with `exec_dynamic=True`.
```

Comment:
The documentation of this function seems to be broken:
![image](https://github.com/user-attachments/assets/3b6f14b4-307d-4cf6-8b84-6e86701a4dc9)
I know it's probably out of scope of this PR, but maybe you could consider fixing it?
Also, the monospace formatting might also apply here.